index.html

```diff
@@ -14,6 +14,10 @@
 		<meta name="color-scheme" content="dark light" />
 		<title>Smart Shopping List</title>
 		<script type="module" src="/src/index.jsx" async></script>
+		<script
```

Comment:
I am going to edit the formatting of this file. It's a bit strange to split the tags like that. I'd just apply the default formatting that VS Code applies.